app/services/rhapsody_realities.js

```diff
@@ -31,6 +31,14 @@ class RhapsodyService {
     createRhapsody(data) {
         let rhapsody = {};
         rhapsody.title = data.title;
+        rhapsody.opening_verse = data.opening_verse;
+        rhapsody.body = data.body;
+        rhapsody.confession = data.confession;
+        rhapsody.further_scripture = data.further_scripture;
+        rhapsody.one_year_bible_plan = data.one_year_bible_plan;
+        rhapsody.date = data.date;
+        // rhapsody.lang = data.lang;
```

Comment:
we usually don't commit commented code,
so please take this off, since we are yet ready for another `lang`

Reply:
coolio